### PR TITLE
Add Initial Product Version Page

### PR DESCRIPTION
**Pull request by gsandok**
Button and resulting page are shown below
Next step is to add dropdown to page that displays tags 
Screenshot 2023-10-26 at 4 07 20 PM
Screenshot 2023-10-26 at 4 07 29 PM

### CODE REVIEW

File: services/frontend-service/src/ui/components/ProductVersion/ProductVersion.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+.page_description {
+    font-style: normal;
+    position: relative;
+    font-weight: 700;
+    line-height: 32px;
+    padding: 10px 40px;
+    color: #4a4a4a;
+    justify-content: space-between;
+    display: flex;
+    background: linear-gradient(var(--mdc-theme-primary), var(--mdc-theme-primary)) left 9px top 14px/2px 100% no-repeat;
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/src/ui/components/ProductVersion/ProductVersion.scss:41]: please check if you can simplify some of this code by use `@extend .text-bold` or something similar.

File: services/frontend-service/src/ui/components/ProductVersion/ProductVersion.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    color: var(--mdc-theme-primary);
+}
+
+.page_description {
+    font-style: normal;
+    position: relative;
+    font-weight: 700;
+    line-height: 32px;
+    padding: 10px 40px;
+    color: #4a4a4a;
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/src/ui/components/ProductVersion/ProductVersion.scss:38]: can we use a default color here? If not, please create a var, and reference it here.

File: services/frontend-service/src/ui/components/ProductVersion/ProductVersion.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Copyright 2023 freiheit.com*/
+import './ProductVersion.scss';
+
+export type ProductVersionProps = {
+    environment: string;
+};
+export const ProductVersion: React.FC<ProductVersionProps> = (props) => {
+    const { environment } = props;
+    // TODO(GS): call tags endpoint
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/src/ui/components/ProductVersion/ProductVersion.tsx:23]: either do the task behind the TODO, or add the story ID here ("will be done in story XYZ")

File: services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx [hunk 4]
```diff
@@ -61,6 +67,13 @@ export const EnvironmentCard: React.FC<{ environment: Environment }> = (props) =
                         icon={<Locks />}
                         onClick={addLock}
                     />
+                    <div>
+                        <Button
+                            className="environment-action"
+                            label={'Display version for ' + environment.name}
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.tsx:73]:
```suggestion
                            label={'Display versions for ' + environment.name}
```
Should be more than one, right?

File: services/frontend-service/src/ui/components/ProductVersion/ProductVersion.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    environment: string;
+};
+export const ProductVersion: React.FC<ProductVersionProps> = (props) => {
+    const { environment } = props;
+    return (
+        <div className="product_version">
+            <h1 className="environment_name">{'Product Version for ' + environment}</h1>
+            <div className="page_description">
+                {
+                    'This page shows the version of the product for the selected environment based on tags to the repository.\nIf there are no tags, then no data can be shown.'
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/src/ui/components/ProductVersion/ProductVersion.tsx:28]: does that `\n` render as expected?

**Review by sven-urbanski-freiheit-com — APPROVED** [review]

File: services/frontend-service/src/ui/components/ProductVersion/ProductVersion.tsx (hunk 5 above)
Comment (gsandok) [reply services/frontend-service/src/ui/components/ProductVersion/ProductVersion.tsx:28]: no it does not, but can be removed

**Review by sven-urbanski-freiheit-com — APPROVED** [review]

**Pull request merged**